grails-app/conf/Config.groovy

```diff
@@ -137,6 +137,8 @@ environments {
         grails.serverURL = "http://${localhostAddress}:8080/$appName"
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "https://catalogue-123.aodn.org.au/geonetwork"
+
+        grails.mail.port = com.icegreen.greenmail.util.ServerSetupTest.SMTP.port
```

Comment:
FWIW, I've also been testing using the same email config as with gogoduck (i.e. sendmail) and it works fine with that.  So we can use the same chef code for this (SES aside).